web/src/admin/stages/authenticator_sms/AuthenticatorSMSStageForm.ts

```diff
@@ -269,6 +237,40 @@ export class AuthenticatorSMSStageForm extends BaseStageForm<AuthenticatorSMSSta
                     ${this.provider === ProviderEnum.Generic
                         ? this.renderProviderGeneric()
                         : this.renderProviderTwillio()}
+                    <ak-form-element-horizontal label=${msg("Mapping")} name="mapping">
+                        <ak-search-select
+                            .fetchObjects=${async (
+                                query?: string,
+                            ): Promise<NotificationWebhookMapping[]> => {
+                                const args: PropertymappingsNotificationListRequest = {
+                                    ordering: "name",
+                                };
+                                if (query !== undefined) {
```

Comment:
Ken and I are on the hunt for comparisons to `undefined` to replace them with falsey, or instead comparing them `null` when an explicit value is set. 

```suggestion
                                if (!query) {
```